README.md

```diff
@@ -1,59 +1,57 @@
 
-### `yarn build`
+## Cloning the repository
 
-Builds the app for production to the `build` folder.<br /> It correctly bundles
-React in production mode and optimizes the build for the best performance.
+You can clone this repository by running:
 
-The build is minified and the filenames include the hashes.<br /> Your app is
-ready to be deployed!
+```
+git clone https://github.com/bench-routes/dashboard.git
+```
 
-See the section about
-[deployment](https://facebook.github.io/create-react-app/docs/deployment) for
-more information.
+## Installing npm dependencies
 
-### `yarn eject`
+The Dashboard depends on few [npm](https://www.npmjs.com/) packages. These are not checked in, so you will need to download and install them locally via the Yarn package manager:
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+    yarn
 
-If you aren’t satisfied with the build tool and configuration choices, you can
-`eject` at any time. This command will remove the single build dependency from
-your project.
+Yarn consults the `package.json` and `yarn.lock` files for dependencies to install. It creates a `node_modules` directory with all installed dependencies.
 
-Instead, it will copy all the configuration files and the transitive
-dependencies (webpack, Babel, ESLint, etc) right into your project so you have
-full control over them. All of the commands except `eject` will still work, but
-they will point to the copied scripts so you can tweak them. At this point
-you’re on your own.
+**NOTE**: Remember to change directory to `dashboard` before running this command and the following commands.
 
-You don’t have to ever use `eject`. The curated feature set is suitable for
-small and middle deployments, and you shouldn’t feel obligated to use this
-feature. However we understand that this tool wouldn’t be useful if you couldn’t
-customize it when you are ready for it.
+## Running a local development server
 
-## Learn More
+You can start a development server for the Dashboard by running:
 
-You can learn more in the
-[Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+    yarn start
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+This will open a browser window with the React app running on http://localhost:3000/. The page will reload if you make edits to the source code. You will also see any lint errors in the console.
```

Comment:
This will open a browser window with the React app running on http://localhost:3000/ in "watch" mode i.e. the web server will reload if you make edits to the source code and you will also see any linting errors in the console.